controllers/openstackdataplanenodeset_controller.go

```diff
@@ -489,6 +603,10 @@ func (r *OpenStackDataPlaneNodeSetReconciler) SetupWithManager(mgr ctrl.Manager)
 			dnsMasqWatcher).
 		Watches(&dataplanev1.OpenStackDataPlaneDeployment{},
 			deploymentWatcher).
+		Watches(&corev1.ConfigMap{},
+			cmVarsWatcher).
```

Comment:
I think we can use a single func for both secrets and config maps.  Also maybe good to just wtach for configmap/secret version changes.